src/store/ducks/plans.ts

```diff
@@ -659,14 +644,14 @@ export const getPlansArrayByParentJurisdictionId = createSelector(
  * @param {Registry} state - the redux store
  * @param {PlanFilters} props - the plan filters object
  */
-export const makePlansArraySelector = () => {
+export const makePlansArraySelector = (planKey?: string) => {
```

Comment:
What happens if a `planKey` is not provided?

Reply:
@cKellyDesign this function would continue to work as before the refactor